plugins/plugin-radius/src/components/applications/ApplicationIcon.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { SvgIcon, SvgIconProps } from '@material-ui/core';
+
+export const ApplicationIcon = (
+  props: SvgIconProps & { shape?: 'full' | 'square' },
+) => {
+  const originalWidth = 1080;
+  const originalHeight = 1080;
+
+  return (
+    <SvgIcon
+      {...props}
+      viewBox={
+        props.shape === 'square'
+          ? `0 0 ${originalHeight} ${originalHeight}`
+          : `0 0 ${originalWidth} ${originalHeight}`
```

Comment:
Do you need all of the complexity here with `.shape`? This was needed for the "Radius" logo because the text is part of the graphic.

Reply:
I kept this for in case we evolve the icons into non-square shapes in the future, but I get your point that it seems unnecessary, so I'll remove the `.shape` property since we don't really need it at this point.